includes/admin/settings/class-settings-advanced.php

```diff
@@ -52,6 +52,28 @@ public function get_settings() {
 							'id'   => 'give_title_data_control_2',
 							'type' => 'title',
 						),
+						array(
+							'name'    => __( 'Cache', 'give' ),
+							'desc'    => __( 'If caching enabled then plugin start caching custom post type related queries and reduce load from your site.', 'give' ),
+							'id'      => 'cache',
+							'type'    => 'radio_inline',
+							'default' => 'enabled',
+							'options' => array(
+								'enabled'  => __( 'Enabled', 'give' ),
+								'disabled' => __( 'Disabled', 'give' ),
+							),
+						),
+						array(
```

Comment:
@ravinderk This settings ( i.e. Remove Data on Uninstall) is duplicated in this PR. This happens when i add this PR to Give `release 2.0`

**Check Screenshot for reference:**
![image](https://user-images.githubusercontent.com/1852711/33661448-ec75f5a4-daad-11e7-9fa9-feb65c88b398.png)